plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -199,8 +210,15 @@ export const SearchModal = (props: SearchModalProps) => {
     >
       {open && (
         <SearchContextProvider inheritParentContextIfAvailable>
-          {(children && children({ toggleModal })) ?? (
-            <Modal toggleModal={toggleModal} />
+          {(children &&
+            children({
+              toggleModal,
+              searchResultChildren: searchResultChildren || [],
```

Comment:
As mentioned early in the comments, this is the case where people are completely customizing the search modal, they will define the search results items manually, so we don't need to forward result items.
```suggestion

```